### ISSUE TITLE
Services & APIs - Edit Properties - click Save ===> Error

### ISSUE THREAD

**Issue by StyleTang** [microcks/microcks#1187]
### Describe the bug

Go to Services & APIs choose one of the APIs, click `Edit Properties`, click `Save`, then you will get 
<img width="1680" alt="image" src="https://github.com/microcks/microcks/assets/5286751/e2db3749-f1fd-480c-959a-54f25e7f4ea1">

### Expected behavior

Save succeed.

### Actual behavior

Error popup

### How to Reproduce?

# How to produce
Just follow https://microcks.io/documentation/getting-started/
```shell
git clone https://github.com/microcks/microcks.git --depth 10
cd microcks/install/docker-compose
docker-compose up -d
```

go to `http://localhost:8080` and login with `Username: admin  Password: microcks123`

Import `API Pastry - 2.0 - 2.0.0 - GET /pastry`
Go to Services & APIs choose one of the APIs, click `Edit Properties`, click `Save`, then you will get 
<img width="1680" alt="image" src="https://github.com/microcks/microcks/assets/5286751/e2db3749-f1fd-480c-959a-54f25e7f4ea1">

GET /pastry Dispatch properties cannot be updated (Http failure response for http://localhost:8080/api/services/66569fb18486c11ffb992daf/operation?operationName=GET%20/pastry: 400 OK)

Go to Log, we can see.
```
2024-05-29 11:56:08 03:56:08.064  WARN 1 --- [080-exec-3] .w.s.m.s.DefaultHandlerExceptionResolver : Resolved [org.springframework.http.converter.HttpMessageNotReadableException: JSON parse error: Cannot deserialize value of type `java.util.ArrayList<io.github.microcks.domain.ParameterConstraint>` from String value (token `JsonToken.VALUE_STRING`)]
```

### Microcks version or git rev

quay.io/microcks/microcks:1.9.1

### Install method (`docker-compose`, `helm chart`, `operator`, `docker-desktop extension`,...)

docker-compose

### Additional information

As mentioned before.

**Comment by lbroudoux** [comment]
Thanks a lot for reporting this issue, I succeeded in reproducing it. I'm currently investigating what could be the root cause.

**Comment by lbroudoux** [comment]
Git it! This is due to a UX refactoring. `operationConstraints` is no longer initialized to empty array when empty but initialized as a string. I think I'll fix this and in a fix release for `1.9.1`. Will do that on this evening I guess.
